x-pack/elastic-agent/pkg/agent/cmd/enroll.go

```diff
@@ -59,6 +60,8 @@ func addEnrollFlags(cmd *cobra.Command) {
 	cmd.Flags().Uint16P("fleet-server-port", "", 0, "Fleet Server HTTP binding port (overrides the policy)")
 	cmd.Flags().StringP("fleet-server-cert", "", "", "Certificate to use for exposed Fleet Server HTTPS endpoint")
 	cmd.Flags().StringP("fleet-server-cert-key", "", "", "Private key to use for exposed Fleet Server HTTPS endpoint")
+	cmd.Flags().StringSliceP("headers", "", []string{}, "App auth token used for elasticsearch")
+	cmd.Flags().StringSliceP("kibana-headers", "", []string{}, "App auth token used for kibana")
```

Comment:
Enroll command never talks to Kibana, you can remove this completely. Only the `container` command, by itself will talk to Kibana.